### ISSUE TITLE
DayOfWeek not precisely defined in modelling spec & schema.org

### ISSUE THREAD

**Issue by Mortuie** [openactive/modelling-opportunity-data#266]
It is unclear whether `dayOfWeek` attribute is typed as a `schema.org/DayOfWeek` or `ArrayOf#schema.org/DayOfWeek`. Neither the modelling spec nor schema.org accurately define this attribute. 

The only place this seems to be defined is in the data models project linked here:
[data model](https://github.com/openactive/data-models/blob/c9217bd582a9f0bab665aecf5c84f3fcabca7977/versions/2.x/models/OpeningHoursSpecification.json#L54) Where it clearly states this typed as: `ArrayOf#https://schema.org/DayOfWeek"`

Moreover in [DayOfWeek](https://schema.org/DayOfWeek). Description: `The day of the week for which these opening hours are valid.` suggests that this is singular of the type `schema.org/DayOfWeek` rather than the array type.

As this isn't clearly defined in Schema.org or in the modelling spec, it would be good to define this field in this [section](https://openactive.io/modelling-opportunity-data/#describing-locations-code-schema-place-code-)

**Comment by nickevansuk** [comment]
Good point @Mortuie! See https://github.com/openactive/modelling-opportunity-data/issues/258 for the origin of this - certainly a good one for the spec to clarify as it's currently under-specified
